app/javascript/components/Legend.jsx

```diff
@@ -39,7 +39,7 @@ class Legend extends React.Component {
 
   render() {
     return (
-        <div className={`row ${this.hidden_class_name()}`}>
+        <div className={`row${this.hidden_class_name()}`}>
```

Comment:
I would include the space so that we're always sure `row` is a valid classname in this context.

Reply:
👍